### PR TITLE
Net Ping: Add new action plugin for *_ping network modules

### PR DESCRIPTION
**Pull request by jmcgill298**
##### SUMMARY

Add new action plugin for network ping modules

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

net_ping
##### ANSIBLE VERSION

```
2.4
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/network/ping/net_ping.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+# (c) 2017, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
```
Comment (Qalthos) [inline lib/ansible/modules/network/ping/net_ping.py]: -> `'supported_by': 'community'`

**Comment by gundalow** [comment]
Hi, 
Can you please move this to `modules/net_tools/`

Also I think this and the ping module's `note:` section should link to the other module with `M(ping) ` stating when each one should be used.

**Review by Qalthos — CHANGES_REQUESTED** [review]
Can I ask why you want this? The only implementation module is presently `nxos_ping`, are you planning on submitting (or expecting to see) a ping module for another platform?

File: lib/ansible/modules/network/ping/net_ping.py (hunk 1 above)
Comment (jmcgill298) [reply lib/ansible/modules/network/ping/net_ping.py]: https://github.com/ansible/ansible/pull/28557

**Comment by gundalow** [comment]
Ignore my comment about net_tools

**Comment by jmcgill298** [comment]
@Qalthos I have updated the "supported_by"

**Comment by Qalthos** [comment]
That being said, we don't need a new directory just for `ping`, modules/network/system is a fine place for this module to live. Sorry for not getting that to you earlier.

**Comment by jmcgill298** [comment]
alright, I will move it. I didn't like that either, but didn't know what else to do with it.

**Comment by jmcgill298** [comment]
@Qalthos any issue with this one now? Do I need to do anything to have this merged with ios_ping?

**Comment by Qalthos** [comment]
Update RETURN so it matches nxos_ping/ios_ping is all you need

**Review by Qalthos — APPROVED** [review]

**Pull request merged**